faker/providers/internet/__init__.py

```diff
@@ -115,6 +115,13 @@ class Provider(BaseProvider):
         '?txtsize=55&txt={width}x{height}&w={width}&h={height}',
         'https://www.lorempixel.com/{width}/{height}',
         'https://dummyimage.com/{width}x{height}',
+        'https://baconmockup.com/{width}/{height}',
+        'https://loremflickr.com/{width}/{height}',
```

Comment:
My understanding is that this service returns random images from flickr, without content curation. I'm not sure I'm feeling comfortable with having random content returned. I'd rather not have it.